CASC/PortfolioMode.cpp

```diff
@@ -632,12 +632,7 @@ void PortfolioMode::runSlice(Options& strategyOpt)
 {
   System::registerForSIGHUPOnParentDeath();
   UIHelper::portfolioParent=false;
-
-  env.timer->reset();
-  env.timer->start();
-
-  Timer::resetInstructionMeasuring();
-  Timer::setLimitEnforcement(true);
```

Comment:
Now unnecessary.

Reply:
But do we then inherit the parents "o'clock" and "instr count"? Because they should start from 0...